src/ingredient/ingredient.repository.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { CreateIngredientBody } from './dto';
+
+@Injectable()
+export class IngredientRepository {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  async createOne(data: CreateIngredientBody) {
```

Comment:
no need for One in method name. the first thing anyone assumes when reading create(), is that it creates single element. It is pretty intuitive. So, no need to attach useless words like, single, one and etc. There are probably cases when that makes sense, but this is not the case